common/strings/stringifier.ts

```diff
@@ -282,3 +284,18 @@ export class StringifiedSet<T> implements Set<T> {
     })();
   }
 }
+
+export class ProtoStringifier<T> implements IStringifier<T> {
```

Comment:
this probably more cleanly belongs inside https://source.cloud.google.com/tada-analytics/github_dataform-co_dataform/+/master:common/protos/index.ts?q=f:protos%20lang:ts too.

Reply:
I did that at first too, but it results in circular dependencies. Shouldn't bother for now I think

Reply:
wait, why? that would imply that `strings/stringifier.ts` somehow depends on `protos/index.ts`, it really shouldn't, how does it?

Reply:
My bad, was making protos depend on itself, not strings